### PR TITLE
Replaced fwd slashes with backslashes for win to fix sass deps watch

### PR DESCRIPTION
**Pull request by lysla**
Running `parcel watch` on Windows wouldn't re-build scss files when only partial sass dependencies, imported in the main .scss file, were modified. The reason is Windows uses forward slashes `/` in the path of these dependencies. #1451

### CODE REVIEW

File: src/assets/SASSAsset.js [hunk 1]
```diff
@@ -65,6 +65,9 @@ class SASSAsset extends Asset {
 
   collectDependencies() {
     for (let dep of this.ast.stats.includedFiles) {
+      if (os.platform() === 'win32') {
```
Comment (DeMoorJasper) [inline src/assets/SASSAsset.js]: What about moving this to:
https://github.com/parcel-bundler/parcel/blob/master/src/Bundler.js#L394
and
https://github.com/parcel-bundler/parcel/blob/master/src/Bundler.js#L704

This way this applies to every asset type, not just scss

Comment (devongovett) [reply src/assets/SASSAsset.js]: maybe you could use `path.normalize` for this https://nodejs.org/api/path.html#path_path_normalize_path

Comment (lysla) [reply src/assets/SASSAsset.js]: I just did another commit, moving the fix in Bundler.js like suggested and using path.normalize. I'm not sure I put it in the right positions, but as far as I could test it, it works. Thanks for your efforts!

**Comment by devongovett** [comment]
actually I believe that since we are calling `fs.realpath` on the path now prior to watching it, that will cause it to be normalized already.

**Comment by lysla** [comment]
Thank you for merging @devongovett keep up the awesome work.

**Pull request merged**